examples/Knex.md

```diff
@@ -13,15 +13,22 @@ const DataLoader = require('dataloader');
 const db = require('./db'); // an instance of Knex client
 
 // The list of data loaders
+// we illustrate different approaches to preserve `ids` ordering
 
 const data = {
   user: new DataLoader(ids => db.table('users')
     .whereIn('id', ids).select()
     .then(rows => ids.map(id => rows.find(x => x.id === id)))),
+    // or
+    // .then(rows => {
+    //   const m = new Map(rows.map(row => [row.id, row]));
+    //   return ids.map(id => m.get(id));
+    // })
 
   story: new DataLoader(ids => db.table('stories')
-    .whereIn('id', ids).select()
-    .then(rows => ids.map(id => rows.find(x => x.id === id)))),
+    .whereIn('id', ids)
+    .orderByRaw('array_position(:ids, id)', { ids })
```

Comment:
I assume this syntax is specific to Postgres, right?

Reply:
..in the case with Postgres, you can also consider the `with ordinality` syntax demonstrated here https://stackoverflow.com/questions/866465/order-by-the-in-value-list/35456954#35456954

Reply:
Thanks good point, I updated it